### PR TITLE
dankomax: created verifyAddProductsToComparisonList test

### PR DESCRIPTION
**Pull request by dankomax**

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/rozetka/components/Header.java [hunk 1]
```diff
@@ -68,4 +70,14 @@ public boolean isLanguageSelected(Language language) {
         return isVisible(
                 String.format("(//li[contains(@class, 'lang__item')]/span[contains(text(),'%s')])[1]", language));
     }
+
+    public int getNumberOfProductsInComparisonList() {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/components/Header.java:74]: rename

File: src/main/java/com/softserveinc/ita/rozetka/components/Header.java [hunk 2]
```diff
@@ -68,4 +70,14 @@ public boolean isLanguageSelected(Language language) {
...
+        return Integer.parseInt(getText("//rz-comparison//rz-icon-counter"));
+    }
+
+    @Step("Comparison list modal: open comparison list")
+    public ComparisonListModal openComparisonList() {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/components/Header.java:79]: rename

File: src/main/java/com/softserveinc/ita/rozetka/data/subcategory/GamersGoodsSubcategory.java [hunk 3]
```diff
@@ -8,8 +8,10 @@
 @RequiredArgsConstructor
 public enum GamersGoodsSubcategory implements ISubcategory {
 
-    MONITORS("monitors");
+    MONITORS("monitors", "Монітори");
 
     @NonNull
     private final String subcategoryXpath;
+    @NonNull
+    private final String subcategoryUkrainianName;
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/data/subcategory/GamersGoodsSubcategory.java:16]: subcategoryNameUa

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/components/Header.java (hunk 1 above)
Comment (dankomax) [reply src/main/java/com/softserveinc/ita/rozetka/components/Header.java]: getComparisonListProductQuantity()

File: src/main/java/com/softserveinc/ita/rozetka/components/Header.java (hunk 2 above)
Comment (dankomax) [reply src/main/java/com/softserveinc/ita/rozetka/components/Header.java]: openComparisonListModal()

File: src/main/java/com/softserveinc/ita/rozetka/data/subcategory/GamersGoodsSubcategory.java (hunk 3 above)
Comment (dankomax) [reply src/main/java/com/softserveinc/ita/rozetka/data/subcategory/GamersGoodsSubcategory.java]: done

File: src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            comparisonItem = comparisonPage.getComparisonItem(i);
+            assertThat(comparisonItem.getItemTitle())
+                    .as("Comparison item title should be in a list of added for comparison product titles")
+                    .isIn(productTitles);
+            softly.assertThat(comparisonItem.getItemTitle())
+                    .as("Comparison %s item title should be equal to product title", i)
+                    .isEqualTo(productTitles.get(comparisonProductQuantity - i));
+            softly.assertThat(comparisonItem.getItemPrice())
+                    .as("Comparison %s item price should be equal to product price", i)
+                    .isEqualTo(productPrices.get(comparisonProductQuantity - i));
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java:72]: extract duplication to variables and re-use

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java (hunk 4 above)
Comment (dankomax) [reply src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java]: int reverseOrderProductNumber = comparisonProductQuantity - i;

File: src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+        ComparisonPage comparisonPage = header
+                .openComparisonListModal()
+                .openComparisonPage(CAMERAS);
+
+        assertThat(comparisonPage.getComparisonItemQuantity())
+                .as("Comparison item quantity should be equal to quantity of products added for comparison")
+                .isEqualTo(comparisonProductQuantity);
+
+        ComparisonItem comparisonItem;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java:60]: why is this needed outside of for() loop?

File: src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        SoftAssertions softly = new SoftAssertions();
+        for (int i = 1; i <= comparisonProductQuantity; i++) {
+            comparisonItem = comparisonPage.getComparisonItem(i);
+            assertThat(comparisonItem.getItemTitle())
+                    .as("Comparison item title should be in a list of added for comparison product titles")
+                    .isIn(productTitles);
+
+            int reverseOrderProductNumber = comparisonProductQuantity - i;
+            softly.assertThat(comparisonItem.getItemTitle())
+                    .as("Comparison %s item title should be equal to product title", i)
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka/ComparisonListTest.java:70]: simplify, i + " comparison item title should ... "

**Review by toleksyn — CHANGES_REQUESTED** [review]
